### DISCUSSION TITLE
Working with file upload.

### DISCUSSION THREAD

**Discussion by ArtMin96** [livewire/livewire#2845] [1. Help] [answered]
There is an option that in addition to the picture, the files can also be previewed. For example, I am adding a video and I do not know how to get the preview.

**Comment by nuernbergerA** [comment, marked as answer]
Publish livewire config
https://www.github.com/livewire/livewire/tree/master/config%2Flivewire.php

Add extension to `preview-mime`

**Reply by ArtMin96** [reply]
Thank you for the quick reply. I am adding mp4 format but it does not show a preview.
This is the code I wrote.

```html
@foreach($files as $file)
    <div class="relative post-image-temporary-url" wire:key="{{ $loop->index }}">
        <div class="shadow cursor-pointer absolute top-0 right-0 p-2 mr-2 mt-2 rounded-full bg-gray-600">
            <svg class="h-6 w-6 text-gray-100" fill="none" viewBox="0 0 24 24" stroke="currentColor" wire:click="remove({{ $loop->index }})">
                <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M6 18L18 6M6 6l12 12"></path>
            </svg>
        </div>

        <img class="object-cover w-full" src="{{ $file->temporaryUrl() }}">
    </div>
@endforeach
```

**Reply by nuernbergerA** [reply]
Wrong HTML tag for images it's `img` but for videos it's `video`

**Reply by ArtMin96** [reply]
ah. How could I miss it. Thank you for the clear answer. I will try now.

**Reply by ArtMin96** [reply]
Working. Thank you so much.
